extensions/amp-live-list/0.1/amp-live-list.js

```diff
@@ -662,6 +744,16 @@ export class AmpLiveList extends AMP.BaseElement {
         `${elem.getAttribute('id')} instead.`);
     return time;
   }
+
+  /**
+   * Checks if the elements top is below the viewport height.
+   *
+   * @return {boolean}
+   */
+  isElementBelowViewport_(element) {
+    const elemTop = element./*REVIEW*/getBoundingClientRect().top;
```

Comment:
Use `Resource` `getLayoutBox`. This is both wrong (because `gBCR` is viewport relative and slow :)